lib/govuk_design_system_formbuilder/builder.rb

```diff
@@ -586,8 +586,8 @@ def govuk_check_box(attribute_name, value, hint_text: nil, label: {}, link_error
     #   = f.govuk_submit "Proceed", prevent_double_click: true do
     #     = link_to 'Cancel', some_other_path, class: 'govuk-button__secondary'
     #
-    def govuk_submit(text = config.default_submit_button_text, warning: false, secondary: false, prevent_double_click: true, validate: false, &block)
-      Elements::Submit.new(self, text, warning: warning, secondary: secondary, prevent_double_click: prevent_double_click, validate: validate, &block).html
+    def govuk_submit(text = config.default_submit_button_text, warning: false, secondary: false, custom_class: nil, prevent_double_click: true, validate: false, &block)
```

Comment:
Can we have a line of explanation for `custom_class` above this method for the rdoc please?